src/storage/v2/mvcc.hpp

```diff
@@ -131,13 +131,17 @@ inline Delta *CreateDeleteDeserializedObjectDelta(Transaction *transaction, std:
                                                   std::string &&ts) {
   transaction->EnsureCommitTimestampExists();
   // Should use utils::DecodeFixed64(ts.c_str()) once we will move to RocksDB real timestamps
-  return &transaction->deltas.use().emplace_back(Delta::DeleteDeserializedObjectTag(), std::stoull(ts), old_disk_key);
+  uint64_t ts_id = 0;
+  std::from_chars(ts.data(), ts.data() + ts.size(), ts_id);
```

Comment:
Maybe create utils function which will convert string_view to uint64_t?
We will be able to use it here and in LabelId also.

Reply:
I can add this